proto/google/events/cloud/storage/v1/data.proto

```diff
@@ -0,0 +1,146 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.events.cloud.storage.v1;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/wrappers.proto";
+
+option csharp_namespace = "Google.Events.Protobuf.Cloud.Storage.V1";
+
+// An object within Google Cloud Storage.
+message StorageObjectData {
+  // Describes the customer-specified mechanism used to store the data at rest.
+  message CustomerEncryption {
+    // The encryption algorithm.
+    string encryption_algorithm = 1;
+
+    // SHA256 hash value of the encryption key.
+    string key_sha256 = 2;
+  }
+
+  // Content-Encoding of the object data, matching
+  // [https://tools.ietf.org/html/rfc7231#section-3.1.2.2][RFC 7231 §3.1.2.2]
+  string content_encoding = 1;
+
+  // Content-Disposition of the object data, matching
+  // [https://tools.ietf.org/html/rfc6266][RFC 6266].
+  string content_disposition = 2;
+
+  // Cache-Control directive for the object data, matching
+  // [https://tools.ietf.org/html/rfc7234#section-5.2"][RFC 7234 §5.2].
+  string cache_control = 3;
```

Comment:
Is this file copied? Is there a reason to skip 4?

Reply:
Yes, it's copied (with some changes) from https://github.com/googleapis/googleapis/blob/master/google/storage/v1/storage_resources.proto

Where changes aren't deliberate and required, I'd prefer to stick to what's there, making it easier to diff.